src/components/Actions/ActionCopyNtoFN.vue

```diff
@@ -21,7 +21,7 @@
   -->
 
 <template>
-	<ActionButton icon="icon-up" @click="copyNtoFN">
+	<ActionButton icon="icon-up" @click="copyNToFN">
```

Comment:
Note: This looks weird but there was a hidden unicode char that broke the handler. Guess how long it took to find it :clown_face: